generator/internal/client_generator.cc

```diff
@@ -343,6 +353,20 @@ Status ClientGenerator::GenerateCc() {
   // clang-format on
 
   for (google::protobuf::MethodDescriptor const& method : methods()) {
+    if (IsBidirStreaming(method)) {
+      CcPrintMethod(method, __FILE__, __LINE__,
+                    R"""(
+std::unique_ptr<::google::cloud::AsyncStreamingReadWriteRpc<
+    $request_type$,
+    $response_type$>>
+$client_class_name$::Async$method_name$(::google::cloud::Options options) {
+  ::google::cloud::internal::OptionsSpan span(
+      ::google::cloud::internal::MergeOptions(std::move(options), options_));
```

Comment:
observation: we do not fully qualify `Options`, `internal::OptionsSpan`, or `internal::MergeOptions` elsewhere in this file.

Reply:
Fixed.